cost/turbonomics/buy_reserved_instances_recommendations/aws/README.md

```diff
@@ -0,0 +1,26 @@
+# Turbonomic Buy Reserved Instances Recommendations AWS
+
+## What it does
+
+The Turbonomic Buy Reserved Instances Recommendations AWS policy utilizes Turbonomic Actions API endpoint (POST `https://turbonomic.com/api/v3/markets/{market_uuid}/actions`)  to provide AWS RI purchase recommendations.
+
+## Functional Details
+
+- The policy queries the /api/v3/markets/{market_uuid}/actions endpoint for the Turbonomic api and based on action will return action details and savings for on-boarded cloud instances
+- The policy will error after a day, the authorization cookie parameter will need to be refreshed and re-run manually
+- there is a need to run the login credentials against the (`https://xxxx.turbonomic.com/api/v3/login`) endpoint to manually receive cookie authorization
+
+### Input Parameters
```

Comment:
I don't know if turbonomic_host parameter should be added in this list

Reply:
Good catch, addressed.